### ISSUE TITLE
POST JSON Array not working

### ISSUE THREAD

**Issue by ReeganExE** [connorjburton/hmac-auth-express#11]
Looks like [src/index.js#L56](https://github.com/connorjburton/hmac-auth-express/blob/8f2786a53e9886d90d24a4c7079e88f4c5fe9a82/src/index.js#L56) has filtered out array. So the below request will fall into `HMAC's did not match`.
```http
POST http://localhost:3000/books/
X-Hub-Signature: HMAC 1596288194660:2251857d0a213f300df0777dfc6deec8bafbc8d6747bb4c64cf3c297fca8dee2

[1, 2, 3, 4]
```

**Comment by connorjburton** [comment]
Thanks for reporting this. Looking into this now.

**Comment by connorjburton** [comment]
Hi,

This is now available on NPM under version `5.0.0`

**Comment by ReeganExE** [comment]
Thanks man.
